examples/plain-javascript/README.md

```diff
@@ -22,7 +22,7 @@ you want. But, as an exercise of this demo, we'll download it ourselves. So:
 1. Click at [cldrjs releases tab](https://github.com/rxaviers/cldrjs/releases).
 1. Download the latest package.
 1. Unzip it.
-1. Move its `dist/` files into `cldrjs` of this directory.
+1. Create a `cldrjs` directory alongside `index.html` and `README.md` and move the cldrjs `dist/` files into it.
```

Comment:
@johnnyreilly, I believe we can improve this sentence for clarity.

@kswedberg do you have a suggestion how we could make this sentence more clear?